noxfile.py

```diff
@@ -185,7 +194,7 @@ def cover(session):
     test runs (not system test runs), and then erases coverage data.
     """
     session.install("coverage", "pytest-cov")
-    session.run("coverage", "report", "--show-missing", "--fail-under=99")
+    session.run("coverage", "report", "--show-missing", "--fail-under=100")
```

Comment:
This PR is currently failing due to coverage. Due to generated tests not having full coverage, we cannot reach 100% coverage which is why we had `--fail-under=99`.

Reply:
`docs-presubmit` is currently failing across other PRs so I'd like to get this PR merged but merging is blocked by this line. I'm going to revert this change for now and we can address it in a future PR :+1: .

```suggestion
    session.run("coverage", "report", "--show-missing", "--fail-under=99")
```

Reply:
Ah. The bot does not like that. I see. Hmmmm :thinking:

Reply:
Fixed